commons/commons-util/src/main/kotlin/ebi/ac/uk/base/StringExt.kt

```diff
@@ -48,3 +48,8 @@ fun String.splitIgnoringEmpty(regex: Regex) = this.split(regex).filter { it.isNo
  * Removes the regex expression matching the string.
  */
 fun String.remove(regex: Regex) = replace(regex, "")
+
+/**
+ * Removes the first regex expression matching the string.
+ */
+fun String.removeFirst(regex: Regex) = replaceFirst(regex, "")
```

Comment:
I'd suggest `removeFirstOcurrence` so it's clear from the name. I'm just saying this because when I read `removeFirst`, the first thing that came to my mind was removing the first character.